lib/github_integration/install-branch.js

```diff
@@ -29,15 +30,18 @@ const takedown = async () => {
   }
 };
 
+const makeTestYML = async (page) => {
+  // ONLY WORKS WITH A NEWLY CREATED PROJECT
```

Comment:
"Newly created project" meaning a project before a repo has been installed? Huh, weird. I can't believe you managed to debug this!

Reply:
Hehe, it took a little time.